Okta.AspNet/AssemblyInfo.cs

```diff
@@ -5,4 +5,8 @@
 
 using System.Runtime.CompilerServices;
 
-[assembly: InternalsVisibleTo("Okta.AspNet.Test")]
+[assembly: InternalsVisibleTo("Okta.AspNet.Test, PublicKey=0024000004800000940000000602000000240000525341310004000001000100c9a1a6c8fc5450"+
```

Comment:
How is the publickey value here deterined?

Reply:
it's a dump of a public part of the key pair, it's added to the PR (`okta.aspnet.pk`).

Reply:
I got it using sn.exe